python/ctsm/site_and_regional/plumber2_surf_wrapper.py

```diff
@@ -45,137 +46,154 @@ def get_parser():
         help="Verbose mode will print more information. ",
         action="store_true",
         dest="verbose",
-        default=False,
     )
 
     parser.add_argument(
-        "--16pft",
-        help="Create and/or modify 16-PFT surface datasets (e.g. for a FATES run) ",
+        "--78pft",
+        help="Create and/or modify 78-PFT surface datasets (e.g. for a non-FATES run) ",
         action="store_true",
-        dest="pft_16",
-        default=True,
+        dest="pft_78",
     )
 
-    return parser
+    parser.add_argument(
+        "--overwrite",
+        help="Overwrite any existing files",
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "--plumber2-sites-csv",
+        help=f"Comma-separated value (CSV) file with Plumber2 sites. Default: {PLUMBER2_SITES_CSV}",
+        default=PLUMBER2_SITES_CSV,
+    )
+
+    return parser.parse_args()
 
 
 def execute(command):
     """
-    Function for running a command on shell.
+    Runs subset_data with given arguments.
     Args:
-        command (str):
-            command that we want to run.
+        command (list):
+            list of args for command that we want to run.
     Raises:
-        Error with the return code from shell.
+        Whatever error subset_data gives, if any.
     """
     print("\n", " >>  ", *command, "\n")
 
-    try:
-        subprocess.check_call(command, stdout=open(os.devnull, "w"), stderr=subprocess.STDOUT)
+    sys.argv = command
+    subset_data.main()
+
 
-    except subprocess.CalledProcessError as err:
-        # raise RuntimeError("command '{}' return with error
-        # (code {}): {}".format(e.cmd, e.returncode, e.output))
-        # print (e.ouput)
-        print(err)
+def is_valid_pft(pft_num):
+    """
+    Given a number, check whether it represents a valid PFT
+    """
+    return pft_num >= 1
```

Comment:
Bare ground should be valid, so change from 1 to 0. Also check that it doesn't exceed max PFT, and that max PFT varies based on whether it's a 16-pft or 78-pft run.

Reply:
Done with b7f7af386.